metabase-db/metabase.db/metabase.db.trace.db

```diff
@@ -0,0 +1,170 @@
+2019-04-26 21:21:14 jdbc[4]: exception
```

Comment:
This does not seem like a useful file to have checked in.

Reply:
This is a debug log. I agree that it should not be included in the repo. I will add a gitignore to this folder and ignore this file.